roles/common/tasks/ipmi.yml

```diff
@@ -6,7 +6,7 @@
   apt: pkg=ipmitool
 
 - name: load kernel ipmi modules
-  modprobe: name={{ item }} state=probe
+  modprobe: name={{ item }} state=absent
```

Comment:
This was giving an error thst probe is an invalid state, the documentation here http://docs.ansible.com/ansible/modprobe_module.html also mentions only absent and present as valid states.